docs/data/joy/customization/theme-tokens/theme-tokens.md

```diff
@@ -3,7 +3,9 @@
 <p class="description">Learn about the two categories of tokens within Joy UI's default theme and how to customize them.</p>
 
 The [W3C Community Group](https://github.com/design-tokens/community-group) defines design tokens as: _"...indivisible pieces of a design system such as colors, spacing, typography scale."_
-Joy UI builds up on this concept to develop its theme, consisting of two categories: low-level and global variant tokens.
+Joy UI builds up on this concept to develop its theme, consisting of two categories: 
+1. [Low-level tokens](#low-level-tokens)
+2. [Global variant tokens](#global-variant-tokens)
```

Comment:
IMO, unordered list is better for this:
```suggestion
- [Low-level tokens](#low-level-tokens)
- [Global variant tokens](#global-variant-tokens)
```

Reply:
Hi. I gave it a numbered list because of the description clearly mentioning *two* categories. 

> to develop its theme, consisting of two categories

An unordered list is also valid if future iterations would make require more than two categories.

Let me know what should I do. 😊

Reply:
An ordered list should be no problem in this case.